README.md

```diff
@@ -1,3 +1,15 @@
 # The DMV
 
 This is the starter repo for the BE Mod1 DMV project.
+
+Describe the steps you took to dig in to this code base. What was your process? If you don’t feel you had a process, define one that you might like to try next time.
+ 
+The first step I took was to look at the code and see what was going on. I then compared the code to the tests to get a sense of what was intentioned. After that I ran the tests to confirm that the existing code worked. I then fixed the code for the tests to pass. 
```

Comment:
This is such a great strategy. The tests act as documentation for the code so that's a great place to start.